src/cursor/cur_hs.c

```diff
@@ -283,6 +296,7 @@ __curhs_set_key(WT_CURSOR *cursor, ...)
     start_ts = WT_TS_NONE;
     counter = 0;
 
+    hs_cursor->flags = 0;
     va_start(ap, cursor);
     arg_count = va_arg(ap, uint32_t);
 
```

Comment:
@tetsuo-cpp - This is the code where we are using argument count.

Reply:
Oh, I see. I didn't realise that part is already in `develop`.